### PR TITLE
Update comments

### PR DESCRIPTION
**Pull request by hswopeams**

### CODE REVIEW

File: contracts/BosonRouter.sol [hunk 1]
```diff
@@ -105,8 +122,9 @@ contract BosonRouter is
 
     /**
      * @notice Pause the Cashier && the Voucher Kernel contracts in case of emergency.
-     * All functions related to creating new batch, requestVoucher or withdraw will be paused, hence cannot be executed.
-     * There is special function for withdrawing funds if contract is paused.
+     * All functions related to creating requestCreateOrder, requestVoucher, redeem, refund, complain, cancelOrFault, 
+     * cancelOrFaultVoucherSet, or withdraw will be paused and cannot be executed.
+     * There is a special function for withdrawing funds if contract is paused.
```
Comment (zajck) [inline contracts/BosonRouter.sol:127]: This probably refers to `withdrawEthOnDisaster` from cashier? Someone might be looking for this function in this file, which is a bit confusing.

File: contracts/BosonRouter.sol [hunk 2]
```diff
@@ -129,18 +148,19 @@ contract BosonRouter is
...
-        uint256 _validTo = metadata[1];
-        uint256 _price = metadata[2];
-        uint256 _depositSe = metadata[3];
-        uint256 _depositBu = metadata[4];
-        uint256 _quantity = metadata[5];
+     * @notice Issuer/Seller offers promise as supply token and needs to escrow the deposit. A supply token is
+     * also known as a voucher set. Payment and deposits are specified in ETH.
+     * @param metadata metadata which is required for creation of a voucher set
+     * Metadata array is used for consistency across the permutations of similar functions.
+     * Some functions require other parameters, and the number of parameters causes sa tack too deep error.
```
Comment (zajck) [inline contracts/BosonRouter.sol:155]: "sa tack" -> "stack".
The same mistake is repeated multiple times below, so I won't tag it everywhere

File: contracts/BosonRouter.sol [hunk 3]
```diff
@@ -179,6 +199,28 @@ contract BosonRouter is
         emit LogOrderCreated(tokenIdSupply, msg.sender, metadata[5], ETHETH);
     }
 
+     /**
+     * @notice Internal function called by other TKNTKN requestCreateOrder functions to decrease code duplication. 
+     * Price and deposits are specified in tokens.
+     * @param _tokenPriceAddress address of the token to be used for the price
+     * @param _tokenDepositAddress address of the token to be used for the deposits
+     * @param _tokensSent total number of tokens sent. Must be equal to seller deposit * quantity
+     * @param deadline deadline after which permit signature is no longer valide. See EIP-2612
```
Comment (zajck) [inline contracts/BosonRouter.sol:208]: "valide" -> "valid".
Also repeats below.

File: contracts/BosonRouter.sol [hunk 4]
```diff
@@ -434,21 +567,21 @@ contract BosonRouter is
...
-     * @notice check if _tokenIdSupply mapped to gate contract, 
-     * if it does, deactivate (user,_tokenIdSupply) to prevent double spending
-     * @param _tokenIdSupply    ID of the supply token
+     * @notice Buyer requests/commits to redeem a voucher and receives Voucher Token in return.
+     * Price and deposit is specified in tokens.
+     * @param _tokenIdSupply ID of the supply token
+     * @param _issuer Address of the issuer of the supply token
+     * @param _tokensSent total number of tokens sent. Must be equal to buyer deposit plus price
+     * @param deadline deadline after which permit signature is no longer valide. See EIP-2612
+     * @param vPrice signature component  used to verify the permit on the price token. See EIP-2612
```
Comment (zajck) [inline contracts/BosonRouter.sol:578]: maybe add "v", "r", "s" as part of definitions
(e.g. `@param vPrice v signature component  used to verify the permit on the price token. See EIP-2612`)
I mean, it is obvious what is should be, just parameter names here are not standard.

File: contracts/BosonRouter.sol [hunk 5]
```diff
@@ -17,6 +17,17 @@ import "./UsingHelpers.sol";
 
 /**
  * @title Contract for interacting with Boson Protocol from the user's perspective.
+ * @notice There are multiple permutations of the requestCreateOrder and requestVoucher functions.
+ * Each function name is suffixed with a payment type that denotes the currency type of the 
+ * payment and deposits. The options are:
+ * 
+ * ETHTH  - Price and deposits are specified in ETH
```
Comment (mischat) [inline contracts/BosonRouter.sol:24]: ETHETH ... missing an E

File: contracts/BosonRouter.sol [hunk 6]
```diff
@@ -89,6 +100,12 @@ contract BosonRouter is
         );
     }
 
+    /**
+     * @notice Construct and initialze the contract. Inizializes associated contract addresses
```
Comment (mischat) [inline contracts/BosonRouter.sol:104]: US spelling :)

Comment (mischat) [reply contracts/BosonRouter.sol:104]: hehe

**Review by mischat — DISMISSED** [review]
This looks good to me

File: contracts/BosonRouter.sol (hunk 5 above)
Comment (hswopeams) [reply contracts/BosonRouter.sol]: Included in next commit.

File: contracts/BosonRouter.sol (hunk 6 above)
Comment (hswopeams) [reply contracts/BosonRouter.sol]: Included in next commit.

File: contracts/BosonRouter.sol (hunk 1 above)
Comment (hswopeams) [reply contracts/BosonRouter.sol]: Included in next commit.

File: contracts/BosonRouter.sol (hunk 2 above)
Comment (hswopeams) [reply contracts/BosonRouter.sol]: Included in next commit.

File: contracts/BosonRouter.sol (hunk 3 above)
Comment (hswopeams) [reply contracts/BosonRouter.sol]: Included in next commit.

File: contracts/BosonRouter.sol (hunk 4 above)
Comment (hswopeams) [reply contracts/BosonRouter.sol]: Included in next commit.

**Review by zajck — APPROVED** [review]

**Pull request merged**